### ISSUE TITLE
Better specs for terra version 

### ISSUE THREAD

**Issue by wenzeslaus** [ncsu-landscape-dynamics/rpops#101]
## Problem description

I'm not sure how exactly the terra versions are numbered, but there is couple of commits in Git which have 1.1-7 in the DESCRIPTION file. Just any of them is not enough. The first commit rpops now works with is https://github.com/rspatial/terra/commit/cd4c35c7818adc26be896d65d130070843fb61c5.

## Expected behavior, suggested solution

Notably, _this will resolve itself over time_ (when the version we need is on CRAN) and I'm not sure what is the right for now.

The terra package is changing rapidly and we are currently using the development version from GitHub, so maybe using specific version in readme which is guaranteed to work?

```
devtools::install_github("rspatial/terra", ref="cd4c35c7818adc26be896d65d130070843fb61c5")
```

And/or using 1.1.8 (1.1-8) in DESCRIPTION file?

On the other hand, in the CI we are simply using the latest version/commit, so with active development in rpops (or a scheduled action we set up), we discover a potential incompatibility with terra here in the repo.

## More info

Currently, you can get things like:

```
> devtools::load_all()
...
Warning message:
object ‘patches’ is not exported by 'namespace:terra' 
> packageVersion("terra")
[1] ‘1.1.7’
```

**Comment by ChrisJones687** [comment]
Terra version 1.1.17 (1.1-17) is now avaliable on CRAN I will update the documentation to make the default branch to also fix the annoying install issue in #98

**Comment by wenzeslaus** [comment]
Jst to understand that better, I tried to figure out from CRAN which commit it actually uses, but I don't see it. Assuming that it works and that transition to 1.1.18 will happen sooner rather than later, I would say we can close this.
